client/src/components/item-details/ItemDetails.tsx

```diff
@@ -28,7 +35,7 @@ export default function ItemDetails() {
     // console.log('PRODUCT:', {...product, size: size})
     // return {...product, size: size}
     if (size && product) {
-      setItem({...product, size: size})
+      // setItem({...product, size: size})
 
       // change this later to useRef
       const sizes = document.querySelectorAll('.size')
```

Comment:
we can't have a mix of jquery and react in the app.

Reply:
We don't, unless somebody else is doing it. Do you see any jquery packages installed?